dev/scripts/up.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
```

Comment:
Its always best to start with `-euo pipefail` to prevent bugs down the line

Reply:
i had it like that, but it was failing for me for some reason. i can try it again